spec/services/create_location_decorator_spec.rb

```diff
@@ -0,0 +1,54 @@
+RSpec.describe CreateLocationDecorator, '.build' do
+  let(:postcode) { 'BT7 3AP' }
+  let(:search_limit) { 3 }
+  let(:booking_location_id) { 222 }
+  let(:booking_location) { double(id: booking_location_id) }
+  let(:location_id) { 111 }
+  let(:location) { double(id: location_id, booking_location_id: booking_location_id) }
+  let(:nearest_locations) { double }
+  let(:search_context) { double }
+  let(:twilio_number) { double }
+  let(:location_decorator) { double }
+
+  def create_decorator
+    described_class.build(location: location,
+                          postcode: postcode,
+                          search_limit: search_limit)
+  end
+
+  subject(:decorator) { create_decorator }
+
+  before do
```

Comment:
I'm on the fence as to whether all this stubbing is good/bad given this is a service object. It feels like we're just testing our stubs. What do you think?